Assets/Scripts/BulletMovement.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BulletMovement : MonoBehaviour
```

Comment:
Comment je class, wat is de functie van deze class? Waar dient deze class voor? Wat gaat deze class uitvoeren?